odoo/models.py

```diff
@@ -5513,64 +5513,73 @@ def filtered(self, func):
         return self.browse([rec.id for rec in self if func(rec)])
 
     def filtered_domain(self, domain):
-        if not domain: return self
+        """Return the records in ``self`` satisfying the domain (in-place).
```

Comment:
I don't understand the meaning of `in-place` in this context.

Reply:
I made a confusing between `stable` and `in-place` :cry: